lib/matplotlib/tests/test_lines.py

```diff
@@ -194,3 +197,24 @@ def test_nan_is_sorted():
 def test_step_markers(fig_test, fig_ref):
     fig_test.subplots().step([0, 1], "-o")
     fig_ref.subplots().plot([0, 0, 1], [0, 1, 1], "-o", markevery=[0, 2])
+
+
+def test_marker_as_markerstyle():
+    fig, ax = plt.subplots()
+    line, = ax.plot([2, 4, 3], marker=MarkerStyle("D"))
+    fig.canvas.draw()
+    assert line.get_marker() == "D"
+
+    #continue with smoke tests:
```

Comment:
There should be a space after `#`; dunno why flake8 isn't complaining.

Reply:
This is because E265 is ignored throughout:
https://github.com/matplotlib/matplotlib/blob/2f85640a2c3a91a39ae5de62c9a574fea06bd929/.flake8#L8